Data/ByteString/Lazy/Internal.hs

```diff
@@ -275,6 +276,17 @@ concat css0 = to css0
     to []               = Empty
     to (cs:css)         = go cs css
 
+-- | Repeats the given ByteString n times.
+times :: Integral a => a -> ByteString -> ByteString
+times 0 _ = Empty
+times _ Empty = Empty
+times n lbs0@(Chunk bs lbs)
+  | n < 0 = error "stimes: non-negative multiplier expected"
```

Comment:
It seems slightly inconsistent to me that duplicating `Empty` with a negative multiplier is allowed. I think it might be better to rule that out.

Reply:
Yes it was inconsistent, I added some guards to throw the same Error message as the other n<0 branch